src/xform_to_datalog/arcs_manifest_tree/handle_connection_spec_test.cc

```diff
@@ -38,18 +38,105 @@ TEST_P(RoundTripHandleConnectionSpecProtoTest,
           original_proto, result_proto));
 }
 
+// Note: each of these protos has a type field that is just a TEXT primitive.
+// Parsing the type correctly isn't the interesting part of parsing a
+// HandleConnectionSpec; that is tested elsewhere. The type field is required
+// and we are checking that it can be made back into an identical proto, so
+// that suffices for this test.
 static const ProtoStringAndExpectations string_and_expections_array[] = {
-    { .proto_str = "name: \"foo\" direction: READS", .expected_name = "foo",
-      .expected_reads = true, .expected_writes = false},
-    { .proto_str = "name: \"bar\" direction: WRITES", .expected_name = "bar",
-      .expected_reads = false, .expected_writes = true },
-    { .proto_str = "name: \"baz\" direction: READS_WRITES", .expected_name =
-    "baz", .expected_reads = true, .expected_writes = true}
+    { .proto_str = "name: \"foo\" direction: READS type: { primitive: TEXT }",
+      .expected_name = "foo", .expected_reads = true, .expected_writes = false},
+    { .proto_str = "name: \"bar\" direction: WRITES type: { primitive: TEXT }",
```

Comment:
May be have a different primitive type?

Reply:
I have only implemented the `TEXT` primitive type at this point; no other primitive type will round trip correctly at this time.